tests/test_ext_ai.py

```diff
@@ -255,20 +257,32 @@ async def test_ext_ai_indexing_02(self):
             # updating an object should make it disappear from results.
             # (the read is done in the same tx, so there is no possible
             # race where the worker picks it up before the read)
-            await self.assert_query_result(
-                """
-                update Stuff filter .content like '%Earth'
-                set { content2 := ' are often grey' };
-                """ + qry,
-                [
-                    {
-                        'content': 'Skies on Mars',
-                        'content2': ' are red',
-                        'distance': 0.4284523933505918,
-                    },
-                ],
-                variables=dict(qv=qv),
-            )
+            async for tr in self.try_until_succeeds(
+                ignore=(
+                    edgedb.TransactionConflictError,
+                    edgedb.TransactionSerializationError,
+                ),
+                timeout=30.0,
+            ):
+                async with tr:
+                    try:
```

Comment:
Do we need this extra `try`/`except` wrapping?

Reply:
oh, lol

Reply:
Definitely makes it easier to find this particular error.